README.md

```diff
@@ -49,7 +49,7 @@ changes in the mesh startup.
 Set the common environment variables used in this document:
 
 ```shell
-export PROJECT_ID=wlhe-cr
+export CLUSTER_PROJECT_ID=wlhe-cr
```

Comment:
I suggest leaving PROJECT_ID as is, to indicate the project where we deploy the workload.

And use CONFIG_PROJECT_ID as a new parameter, defaulting to PROJECT_ID.